### ISSUE TITLE
Tyto Calls Not via attributes

### ISSUE THREAD

**Issue by JMante1** [SynBioDex/tyto#39]
Currently you need to call SO.promoter. It would be very helpful if there was a function tyto(ontology name, ontology term) which returns the uri. This is as getattr(variable1, variable2) doesn't work.

**Comment by bbartley** [comment]
Something like this is currently supported:
```
tyto.endpoint.Ontobee.get_uri_by_term(tyto.SO, 'promoter')
```
where the first argument is a `tyto.Ontology` object.

**Comment by JMante1** [comment]
Okay, so could use tyto.endpoint.Ontobee.get_uri_by_term(getattr(tyto, 'SO'), 'promoter')?

**Comment by bbartley** [comment]
I think that call would work, although generally it's easier to just use the `SO` instance variable directly rather than `getattr`.

**Comment by JMante1** [comment]
Yes, but we want to call the SO term from a variable which is why we use the getattr function

**Comment by JMante1** [comment]
There is something odd happening where tyto.endpoint.Ontobee.get_uri_by_term(getattr(tyto, 'SO'), 'promoter') returns http://purl.obolibrary.org/obo/SO_0000167 instead of https://identifiers.org/SO:0000167

**Comment by JMante1** [comment]
which you get when you try SO.promoter. Why is this the case?
